config/Cargo.toml

```diff
@@ -21,4 +21,4 @@ url = { version = "2.5.0", features = ["serde"] }
 clap = { workspace = true, optional = true } 
 
 [features]
-clap_derive = ["clap"]
+clap_derive = ["clap"]
```

Comment:
it's best to checkout `/config/*` to match main

Reply:
My comment is about the whole directory, notice there're toml files recommitted as well as a backup mod.rs

Reply:
Ugh, I did checkout the whole directory, not sure why git didn't clear deleted files...